Yosemite/Yosemite/Model/WooPlans/WooPlan.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import SwiftUI
```

Comment:
Question: Should Yosemite know about SwiftUI? Or should we add a layer here so we're not leaking presentation/UI details into the business logic. Perhaps we can move this to WooFoundation?

Reply:
No strict rule on it, but I agree and would prefer if it didn't know about SwiftUI. However, WooFoundation doesn't seem like the right place for this file either – it's a model object. 

Ideally, we'd be able to use Color (and Image, and UIColor, and UIImage,) without importing a UI framework, but... we can't. On balance I decided to keep this import here rather than put the code in a strange place, or the properties in simpler primitives that we then convert at runtime.